### PR TITLE
MM-T2372 Deleting: Thread does not move when deleting post

### PR DESCRIPTION
**Pull request by yulyanaR**
Summary

Cypress test: MM-T2372 Deleting

Ticket Link

https://automation-test-cases.vercel.app/test/MM-T2372

### CODE REVIEW

**Review by hmhealey — APPROVED** [review]
Thanks for the PR, @yulyanaR. Looks good to me!

File: e2e/cypress/integration/scroll/deleting_scroll_spec.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
```
Comment (josephbaylon) [inline e2e/cypress/integration/scroll/deleting_scroll_spec.js:10]: Please remove for now.

```suggestion
```

File: e2e/cypress/integration/scroll/deleting_scroll_spec.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+// Group: @scroll
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Deleting', () => {
```
Comment (josephbaylon) [inline e2e/cypress/integration/scroll/deleting_scroll_spec.js:15]:
```suggestion
describe('Scroll', () => {
```

File: e2e/cypress/integration/scroll/deleting_scroll_spec.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                otherUser = user2;
+                cy.apiAddUserToTeam(team.id, otherUser.id).then(() => {
+                    cy.apiAddUserToChannel(testChannelId, otherUser.id);
+                });
+            });
+            cy.visit(testChannelLink);
+        });
+    });
+
+    it('MM-T2372 Deleting', () => {
```
Comment (josephbaylon) [inline e2e/cypress/integration/scroll/deleting_scroll_spec.js:40]:
```suggestion
    it('MM-T2372 Post list does not scroll when the offscreen post is deleted', () => {
```

**Review by josephbaylon — CHANGES_REQUESTED** [review]
Overall looks good. Just minor corrections below.

File: e2e/cypress/integration/scroll/deleting_scroll_spec.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+            Cypress._.times(30, (postIndex) => {
+                cy.postMessageAs({sender: otherUser, message: `Other users p-${postIndex}`, channelId: testChannelId});
+            });
+
+            // # Scroll above the last few messages
+            cy.get('div.post-list__dynamic', {timeout: TIMEOUTS.ONE_SEC}).should('be.visible').
+                scrollTo(0, '90%', {duration: TIMEOUTS.ONE_SEC}).
+                wait(TIMEOUTS.ONE_SEC);
+
+            // * Get the text of the first visible post
```
Comment (josephbaylon) [inline e2e/cypress/integration/scroll/deleting_scroll_spec.js:55]:
```suggestion
            // # Get the text of the first visible post
```

File: e2e/cypress/integration/scroll/deleting_scroll_spec.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            cy.get('.post-message__text:visible').first().then((postMessage) => {
+                firstPostBeforeScroll = postMessage.text();
+            });
+
+            // * Get the text of the last visible post
```
Comment (josephbaylon) [inline e2e/cypress/integration/scroll/deleting_scroll_spec.js:60]:
```suggestion
            // # Get the text of the last visible post
```

File: e2e/cypress/integration/scroll/deleting_scroll_spec.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+                lastPostBeforeScroll = postMessage.text();
+            });
+
+            // # Remove multiline message from the other user
+            cy.externalRequest({user: otherUser, method: 'DELETE', path: `posts/${multilineMessageID}`});
+
+            // # Wait for the message to be deleted
+            cy.wait(TIMEOUTS.ONE_SEC);
+
+            // Verify the first post is the same after deleting
```
Comment (josephbaylon) [inline e2e/cypress/integration/scroll/deleting_scroll_spec.js:71]:
```suggestion
            // * Verify the first post is the same after deleting
```

File: e2e/cypress/integration/scroll/deleting_scroll_spec.js [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+            cy.get('.post-message__text:visible').first().then((firstPostAfterScroll) => {
+                expect(firstPostAfterScroll.text()).equal(firstPostBeforeScroll);
+            });
+
+            // Verify the last post is the same after deleting
```
Comment (josephbaylon) [inline e2e/cypress/integration/scroll/deleting_scroll_spec.js:76]:
```suggestion
            // * Verify the last post is the same after deleting
```

**Review by josephbaylon — CHANGES_REQUESTED** [review]
Please fix step and assertion indicators

**Comment by yulyanaR** [comment]
> Please fix step and assertion indicators

Thanks, it's been updated.

**Review by josephbaylon — APPROVED** [review]
LGTM. Test passed locally. Thanks @yulyanaR !!

**Review by furqanmlk — APPROVED** [review]
Locally passed Thank you
image

**Pull request merged**